volatility3/framework/symbols/linux/elf.json

```diff
@@ -958,7 +1086,7 @@
   },
   "metadata": {
     "producer": {
-      "version": "0.0.1",
+      "version": "0.0.2",
       "name": "ikelos-by-hand",
       "datetime": "2019-10-21T22:52:00"
```

Comment:
If you're modifying this, could you please update the datetime stamp to something more accurate (in case we ever come to rely on it in the future).